boot/zephyr/Kconfig

```diff
@@ -58,7 +58,7 @@ config NRF_CC310_BL
 
 menu "MCUBoot settings"
 
-config SINGLE_IMAGE_DFU
+config SINGLE_APPLICATION_SLOT
 	bool "Single image application"
```

Comment:
Maybe change the title here to be also more in line with the function the config enables.

Reply:
Is it better now?

Reply:
.Kinda sounds like there is a feature where application may span multiple slots.

Reply:
Actually this was same before.

Reply:
`single slot bootloader`?

Reply:
I think that this sounds better.